test/parallel/test-tick-processor.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+var fs = require('fs');
+var assert = require('assert');
+var path = require('path');
+var cp = require('child_process');
+var common = require('../common');
+
+common.refreshTmpDir();
+
+process.chdir(common.tmpDir);
+cp.execFileSync(process.execPath, ['-prof', '-pe',
+    'function fib(n) { return n < 2 ? n : fib(n-1) + fib(n-2); }; fib(40);']);
+var matches = fs.readdirSync(common.tmpDir).filter(function(file) {
+  return /^isolate-/.test(file);
+});
+if (matches.length != 1) {
+  assert.fail('There should be a single log file.');
```

Comment:
This could be replaced with `assert.equal(matches.length, 1, 'There should be a single log file.')` but that's nit-picking.